techmatters/serverless
# End Chat and Post Survey calls modified to allow webchat these actions

## Description
- Guest token set to allow postSurveyInit for webchat clients
- Adds `isTaskStageAssigned` and `taskSid` as a response field for endChat to allow webchat to trigger post survey.

### Checklist


### Related Issues
Fixes # [CHI-1402](https://bugs.benetech.org/browse/CHI-1402)

### Verification steps
- Ensure this webchat [PR](https://github.com/techmatters/webchat/pull/133) is running as well
